### ISSUE TITLE
Bump fabric dependency from ^4.2.0 to ^5.2.1 (latest)

### ISSUE THREAD

**Issue by ajssd** [nhn/tui.image-editor#770]
Please bump the "fabric" dependency from ^4.2.0 to ^5.2.1 (latest).

Fabric v4 relies on an old version of JSDOM, which in turn has dependencies on unsupported packages like request.js and its various dependencies in turn.

**Comment by lja1018** [comment]
@ajssd 
Thank you for your suggestion. I'll update it.

**Comment by mroest** [comment]
Can this be updated? JSON has a security vulnerability See: https://github.com/advisories/GHSA-f4c9-cqv8-9v98

**Comment by aterlamia** [comment]
Yeah could be great if this could be updated as adding a resolution in my package.json does not work because of back ward compatibility issues

**Comment by lja1018** [comment]
@mroest @aterlamia 
There are no updates yet. I'm sorry.
I understand the need for updates. But I don't have enough time. 😢
